java/arcs/android/host/parcelables/ParcelableParticle.kt

```diff
@@ -13,48 +13,48 @@ package arcs.android.host.parcelables
 
 import android.os.Parcel
 import android.os.Parcelable
-import arcs.core.data.ParticleSpec
+import arcs.core.data.Particle
```

Comment:
I really don't like this rename. This class is definitely not a Particle, it does not have any lifecycle, it is pure metadata, so when developers write code and use IDEs, they'll end up with both classes in autocomplete. We had the same situation in TS with identical names in src/runtime and src/runtime/recipe and I found it very confusing and frustrating .

This class's function is to represent a configuration for finding and instantiating a Particle, maybe 'Spec' is the wrong suffix, but I wouldn't call it 'Particle', maybe 'ParticleConfig', or 'ParticleInfo'?

Reply:
Moving to Plan.Particle, as suggested by @shans on chat.